packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -262,6 +262,7 @@ abstract contract StandardBridge is Initializable {
     {
         require(paused() == false, "StandardBridge: paused");
         require(isCustomGasToken() == false, "StandardBridge: cannot bridge ETH with custom gas token");
+        require(msg.value != 0, "StandardBridge: cannot bridge 0 ETH");
```

Comment:
Any reason this was added?

Reply:
I don't think we need this check

Reply:
I know its weird to call this function with 0 value but there is nothing security critical about this check